src/Zynga/Framework/Service/V2/Base.hh

```diff
@@ -14,4 +15,6 @@ abstract class Base extends ServiceBase {
 
   abstract public function response(): ResponseInterface;
 
+  abstract public function server(): ServerInterface;
```

Comment:
I would vote for a more descriptive name than 'server'. Eg ServerGlobals, or ServerVars. Server sounds like it contains more functionality than a static map.

Reply:
Why was this resolved? The name was not changed or a follow up comment was not added.